scss/_patterns_navigation.scss

```diff
@@ -245,19 +263,40 @@ $lightness-threshold: 70;
         margin-right: map-get($grid-margin-widths, medium);
       }
 
-      @media (min-width: $breakpoint-navigation-threshold) {
-        // align baselines of menu items and input text
-        $input-gap-top: $spv-inner--large - $spv-nudge;
+      // @media (min-width: $breakpoint-navigation-threshold) {
```

Comment:
I commented this part out temporarily. I added a new class `.search-box-full-width` to make the search bar have the full width when it has this class and expected the media query overrides the existing one(the commented out part) but it doesn't override it for some reason..